applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_jump_process_2d.py

```diff
@@ -0,0 +1,57 @@
+import KratosMultiphysics
+import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
+
+def Factory(settings, Model):
+    if(not isinstance(settings, KratosMultiphysics.Parameters)):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string")
+
+    return ComputeLiftJumpProcess2D(Model, settings["Parameters"])
+
+# TODO Implement this process in C++ and make it open mp parallel to save time selecting the wake elements
+class ComputeLiftJumpProcess2D(KratosMultiphysics.Process):
+    def __init__(self, Model, settings):
+        # Call the base Kratos process constructor
+        KratosMultiphysics.Process.__init__(self)
+
+        # Check default settings
+        default_settings = KratosMultiphysics.Parameters(r'''{
+            "model_part_name": "",
+            "velocity_infinity": [1.0,0.0,0],
+            "create_output_file": false
+        }''')
+        settings.ValidateAndAssignDefaults(default_settings)
+
+        body_model_part_name = settings["model_part_name"].GetString()
+        if body_model_part_name == "":
+            err_msg = "Empty model_part_name in DefineWakeProcess2D\n"
```

Comment:
Here you can update the error message.
```suggestion
            err_msg = "Empty model_part_name in ComputeLiftJumpProcess2D\n"
```

Reply:
Done as well!